assignment/fish-assignment.Rmd

```diff
@@ -42,18 +50,37 @@ head(ram$area)
 
 ```
 
-
+Side Note: You may notice the `lapply` function above. This function applies a given function
```

Comment:
Yeah, I was thinking about this too.  It's definitely an interesting exercise to attempt the whole exercise with only the R commands we've officially introduced.  I wonder if we should just scrap the `lapply` and manually read in a selection of tables?

One reason to do so would be that we could omit the `*_view` tables, which are essentially pre-joined tables (though not sure if the `units` data shows up anywhere else, would have to check).  Students work with the `values_views` table quite a bit instead of joining the various metadata tables against the `timeseries` table...

Reply:
I agree, I think manually reading in a selection of tables we are interested in has the added benefit of making it much more which tables we want them to join and easier to see what columns are where and what the join is actually accomplishing. I saw a handful of students today with strange duplicates or missing values because their join was incorrect in the beginning. More focus and hand-holding up front, especially if this is the 2nd module, is going to make everything work more smoothly and I think the students will get more out of it. Right now, the students who are encountering joins for the first time don't really have a sense of what they should expect to see as a result and thus are causing problems for themselves down the line.